.htaccess

```diff
@@ -69,6 +69,8 @@ deny from env=stayout
 
 	# redirect all trafic to https
 	# Header set Strict-Transport-Security "max-age=31536000; includeSubDomains" env=HTTPS
+	# Header always edit Set-Cookie (.*) "$1; HTTPOnly; Secure"
```

Comment:
This is the default when you set cookies with fork, but you can choose other options there, won't this prevent people from setting the cookies the way they want?